test/cases_matching_callbacks/1.callbacks.c

```diff
@@ -0,0 +1,7 @@
+#include <metababel/metababel.h>
+
+static void usr_foo_event_callback(void *btx_handle, void *usr_data, uint64_t pf_1, const char *cf_1) {}
```

Comment:
Should we verify that we have the correct data?

Reply:
I was expecting the help of the compiler, that's why I have two types that can not be cast implicitly (`uint64_t `,`const char *`). But yes, I can push the same data for validation.